

<html>
<head>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" >

  <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:700|Pontano+Sans' rel='stylesheet' type='text/css'>
  <link rel="stylesheet" type="text/css" href="course.css">
  <title>RECOMB 2014 - April 2-5, Pittsburgh, Pennsylvania</title>
</head>
<body>

<!-- ========================================================================= -->
<header id="masthead">
<img src="748x205_level_header_city2.JPG" width="935" height="256">
</header>

<!-- ========================================================================= -->
<aside id="sidebar">
    <section>
      <nav id="global">
       <ul>
          <li><a href="index.html">Home</li>
          <li><a href="geninfo.html">General Information</a></li>
          <li><a href="http://en.wikipedia.org/wiki/Alumni_Hall_%28University_of_Pittsburgh%29">Conference Venue</a></li>
          <li><a href="program.html">Program</a></li>
          <li><a href="registration.html">Registration</a></li>
          <li><a href="travel.html">Travel & Visa</a></li>
  <li><a href="lodging.html">Accommodations</a></li>
          <li><a href="travelfellowships.html">Travel Fellowships</a></li>
          <li><a href="keynotes.html">Keynote Speakers</a></li>
          <li><a href="acceptedpapers.html">Accepted Papers</a></li>
          <li><a href="posters.html">Poster Sessions</a></li>
<li><a href="keydates.html">Key Dates</a></li>
          <li><a href="satellites.html">Satellite Workshops</a></li>
          <li><a href="callforpapers.html">Call for Papers</a></li>
          <li><a href="callforposters.html">Call for Posters</a></li>
          <li><a href="callforhighlights.html">Call for Highlights</a></li>
          <li><a href="submission.html">Submission</a></li>
          <li><a href="about.html">About RECOMB</a></li>
          <li><a href="http://www.cmu.edu/ua/pittsburgh/index.html">About Pittsburgh</a></li>
          <li><a href="committees.html">Committees</a></li>
          <li><a href="sponsorship.html">Sponsorship Opportunities</a>
<li><a href="sponsors.html">Sponsors</a></li>
          <li><a href="contact.html">Contact</a></li>
        </ul>
      </nav>
    </section>
</aside>

<div id="content">

<!-- ========================================================================= -->



<div id="content">
<h1>Accepted Papers for RECOMB 2014</h1>

<h2>
The conference proceedings <a href="http://link.springer.com/openurl.asp?genre=issue&issn=0302-9743&volume=8394">(<i>Lecture Notes in Computer Science</i> Volume 8394, 2014)</a> are now online.
</h2>
<p>

<li>Jianzhu Ma, Sheng Wang, Zhiyong Wang and Jinbo Xu.<i> MRFalign: Protein Homology Detection through Alignment of Markov Random Fields</i>
<li>Siavash Mirarab, Nam-Phuong Nguyen and Tandy Warnow. <i>PASTA: ultra-large multiple sequence alignment</i>
<li>Wen-Yun Yang, Farhad Hormozdiari, Eleazar Eskin and Bogdan Pasaniuc. <i>A Spatial-Aware Haplotype Copying Model with Applications to Genotype Imputation</i>
<li>Emily Berger, Deniz Yorukoglu and Bonnie Berger. <i>HapTree: A novel Bayesian framework for single individual polyplotyping using NGS data</i>
<li>Ngan Nguyen, Glenn Hickey, Daniel Zerbino, Brian Raney, Dent Earl, Joel Armstrong, David Haussler and Benedict Paten. <i>Building a Pangenome Reference for a Population</i>
<li>Zhanyong Wang, Jae-Hoon Sul, Sagi Snir, Jose A. Lozano and Eleazar Eskin. <i>Gene-Gene Interactions Detection Using A Two-stage Model</i>
<li>Arne Reimers, Frank Bruggeman, Brett Olivier and Leen Stougie. <i>Fast Flux Module Detection using Matroid Theory</i>
<li>Shaun Mahony, Matthew Edwards, Esteban Mazzoni, Richard Sherwood, Akshay Kakumanu, Carolyn Morrison, Hynek Wichterle and David Gifford. <i>An integrated model of multiple-condition ChIP-seq data reveals predeterminants of Cdx2 binding</i>
<li>Rui Wang and Scott Schmidler. <i>Bayesian Multiple Protein Structure Alignment</i>
<li>Jianling Zhong, Todd Wasson and Alexander Hartemink. <i>Learning protein-DNA interaction landscapes by integrating experimental data through computational models</i>
<li>Ewa Szczurek and Niko Beerenwinkel. <i>Modeling mutual exclusivity of cancer mutations</i>
<li>Marinka Zitnik and Blaz Zupan. <i>Imputation of Quantitative Genetic Interactions in Epistatic MAPs by Interaction Propagation Matrix Completion</i>
<li>Hamidreza Chitsaz and Mohammad Aminisharifabad.<i> Exact Learning of RNA Energy Parameters From Structure</i>
<li>Shutan Xu, Shuxue Zou and Lincong Wang.<i> A geometric clustering algorithm and its applications to structural data</i>
<li>Armin Töpfer, Tobias Marschall, Rowena A Bull, Fabio Luciani, Alexander Schönhuth and Niko Beerenwinkel. <i>Viral quasispecies assembly via maximal clique enumeration</i>
<li>Yana Safonova, Anton Bankevich and Pavel Pevzner. <i>DipSPAdes: Assembler for Highly Polymorphic Diploid Genomes</i>
<li>Jan Hoinka, Alexey Berezhnoy, Zuben E. Sauna, Eli Gilboa and Teresa Przytycka.<i> AptaCluster - A Method to Cluster HT-SELEX Aptamer Pools and Lessons from its Application</i>
<li>Mingfu Shao, Yu Lin and Bernard Moret. <i>An Exact Algorithm to Compute the DCJ Distance for Genomes with Duplicate Genes</i>
<li>Kelley Harris, Sara Sheehan, John Kamm and Yun S. Song. <i>Decoding Coalescent Hidden Markov Models in Linear Time</i>
<li>Chen-Ping Fu, Vladimir Jojic and Leonard Mcmillan. <i>An Alignment-Free Regression Approach for Estimating Allele-Specific Expression using RNA-Seq Data</i>
<li>Henry C.M. Leung, S.M. Yiu and Francis Chin. <i>IDBA-MTP: A Hybrid MetaTranscriptomic Assembler Based on Protein Information</i>
<li>Arun Konagurthu, Parthan Kasarapu, Lloyd Allison, James Collier and Arthur Lesk.<i> On sufficient statistics of least-squares superposition of vector sets</i>
<li>Yu Zheng and Louxin Zhang. <i>Reconciliation with Non-binary Gene Trees Revisited</i>
<li>Keith Noto, Carla Brodley, Saeed Majidi, Diana Bianchi and Donna Slonim. <i>CSAX: Characterizing Systematic Anomalies in eXpression Data</i>
<li>Adrian Guthals, Christina Boucher and Nuno Bandeira. <i>The generating function approach for peptide identification in spectral networks</i>
<li>Raunak Shrestha, Ermin Hodzic, Jake Yeung, Kendric Wang, Thomas Sauerwald, Phuong Dao, Shawn Anderson, Himisha Beltran, Mark A. Rubin, Colin Collins, Gholamreza Haffari and S. Cenk Sahinalp.<i> HIT.nDRIVE: Multi-Driver Gene Prioritization based on Hitting Time</i>
<li>Murray Patterson, Tobias Marschall, Nadia Pisanti, Leo van Iersel, Leen Stougie, Gunnar W. Klau and Alexander Schoenhuth. <i>WhatsHap: Haplotype Assembly for Future-Generation Sequencing Reads</i>
<li>Rayan Chikhi, Antoine Limasset, Shaun Jackman, Jared Simpson and Paul Medvedev.<i> On the representation of de Bruijn graphs</i>
<li>Adam Bloniarz, Ameet Talwalkar, Jonathan Terhorst, Michael Jordan, David Patterson, Bin Yu and Yun Song.<i> Changepoint Analysis for Efficient Variant Calling</i>
<li>Y. William Yu, Deniz Yorukoglu and Bonnie Berger.<i> Traversing the k-mer landscape of NGS read datasets for quality score sparsification</i>
<li>Hetu Kamisetty, Bornika Ghosh, Christopher James Langmead and Chris Bailey-Kellogg.<i> Learning Sequence Determinants of Protein:protein Interaction Specificity with Sparse Graphical Models</i>
<li>Ben Raphael and Fabio Vandin. <i>Simultaneous Inference of Cancer Pathways and Tumor Progression from Cross-Sectional Mutation Data</i>
<li>Shay Zakov and Vineet Bafna.<i> Reconstructing Breakage Fusion Bridge architectures using noisy copy numbers</i>
<li>Derek Aguiar, Eric Morrow and Sorin Istrail.<i> Tractatus: an exact and subquadratic algorithm for inferring identity-by-descent multi-shared haplotype tracts</i>
<li>Hua Wang, Heng Huang and Chris Ding.<i> Correlated Protein Function Prediction via Maximization of Data-Knowledge Consistency</i>
