
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" >

  <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:700|Pontano+Sans' rel='stylesheet' type='text/css'>
  <link rel="stylesheet" type="text/css" href="course.css">
  <title>RECOMB 2014 - April 2-5, Pittsburgh, Pennsylvania</title>
</head>
<body>

<!-- ========================================================================= -->
<header id="masthead">
<img src="748x205_level_header_city2.JPG" width="935" height="256">
</header>

<!-- ========================================================================= -->
<aside id="sidebar">
    <section>
      <nav id="global">
       <ul>
          <li><a href="index.html">Home</li>
          <li><a href="geninfo.html">General Information</a></li>
          <li><a href="http://en.wikipedia.org/wiki/Alumni_Hall_%28University_of_Pittsburgh%29">Conference Venue</a></li>
          <li><a href="program.html">Program</a></li>
          <li><a href="registration.html">Registration</a></li>
          <li><a href="travel.html">Travel & Visa</a></li>
  <li><a href="lodging.html">Accommodations</a></li>
          <li><a href="travelfellowships.html">Travel Fellowships</a></li>
          <li><a href="keynotes.html">Keynote Speakers</a></li>
          <li><a href="acceptedpapers.html">Accepted Papers</a></li>
          <li><a href="posters.html">Poster Sessions</a></li>
<li><a href="keydates.html">Key Dates</a></li>
          <li><a href="satellites.html">Satellite Workshops</a></li>
          <li><a href="callforpapers.html">Call for Papers</a></li>
          <li><a href="callforposters.html">Call for Posters</a></li>
          <li><a href="callforhighlights.html">Call for Highlights</a></li>
          <li><a href="submission.html">Submission</a></li>
          <li><a href="about.html">About RECOMB</a></li>
          <li><a href="http://www.cmu.edu/ua/pittsburgh/index.html">About Pittsburgh</a></li>
          <li><a href="committees.html">Committees</a></li>
          <li><a href="sponsorship.html">Sponsorship Opportunities</a>
<li><a href="sponsors.html">Sponsors</a></li>
          <li><a href="contact.html">Contact</a></li>
        </ul>
      </nav>
    </section>
</aside>

<div id="content">

<!-- ========================================================================= -->



<div id="content">
<h1>Keynote Speakers</h1>
<table>
<tr><a name="Baldwin"><td><h1><img src="keynotes/Baldwin.jpg"></a>
<td><b>Ian T. Baldwin</b><br>
Max Planck Institute for Chemical Ecology, Germany<br>
<a href="http://www.ice.mpg.de/ext/hopa.html?pers=iaba2016&d=itb">Homepage</a></h1>
<tr><td colspan=2><h4>Using our biological legacy before it's lost; melding computational and natural history approaches to provide economic arguments for conservation</h4>
By the end of the century, extinction will be the likely fate for most of the non-domesticated species on the planet, because organisms require habitats. Homo sapiens currently command the lion's share of net carbon fixed, ice-free land and fresh water on the planet as a consequence of our 7 billion plus members, a number that represents half of the total that ever lived. While humans have currently attained "peak baby", with the number of children produced per female has peaked and is declining (coinciding with the attainment of "peak oil"), an additional 3 billion humans will be added to the planet by the end of the century as a consequence of the demographic inertia of the existing population (1).
<br>Under such pressure from our resource-hungry species, it's hard to imagine that many of the other of the planet's inhabitants will remain at the end of the century unless they either directly feed, clothe, house or provide fuel for humans, or parasitize them and our domesticated species. Native species require native habitats, and with 10 billion human mouths to feed, the reasons for preservation must be compelling. 
<br>This talk will describe two decades of research into how a native tobacco plant, <i>Nicotiana attenuata</i>, that lives in the Great Basin Desert of the SW USA has been developed into a model system for the study of all types of plant-ecological interactions, particularly those biotic interactions that dominate the agricultural niche. This plant recognizes attack from specific herbivore species by the particular chemistry of the herbivore's saliva, and uses this recognition to tailor a complicated 6-layered defense response that requires a remodeling of the plant's transcriptome, metabolome and proteome, as well as some of its life history traits.  The science writer, Michael Pollan, inverted the relationship between humans and their domesticated plants to argue that it was plants that domesticated humans, and not vice versa. <i>Nicotiana attenuata</i> has had designs more Machiavellian than domestication for the heterotrophs that feed on it.  
<br>As the genome of <i>Nicotiana attenuata</i> is being sequenced and assembled, forward genetic approaches are being planned to complement the reverse genetic approaches that have been our laboratory's main approach, and the computational approaches that we have developed to identify the genetic resources of this plant's remarkable ecological sophistication will be presented. 
<br>1) http://www.ted.com/talks/hans_rosling_religions_and_babies.html
</tr>

<tr><td height="425" valign="bottom"><h1><a name="Butte"><img src="keynotes/Butte.jpg"></a>
<td><b>Atul Butte, M.D. Ph.D.</b><br>
Division Chief and Associate Professor, Stanford University School of Medicine and Lucile Packard Children's Research Hospital<br>
<a href="http://med.stanford.edu/profiles/Atul_Butte">Homepage</a></h1>
</tr>
<tr><td colspan=2><h4>Translating a trillion points of data into therapies, diagnostics, and new insights into disease</h4>
There is an urgent need to translate genome-era discoveries into clinical
utility, but the difficulties in making bench-to-bedside translations have
been well described. The nascent field of translational bioinformatics may
help.  Dr. Butte's lab at Stanford builds and applies tools that convert
more than a trillion points of molecular, clinical, and epidemiological
data -- measured by researchers and clinicians over the past decade and
now commonly called "big data"-- into diagnostics, therapeutics, and new
insights into disease.  Several of these methods or findings are spun out
into new biotechnology companies.  Dr. Butte, a bioinformatician and
pediatric endocrinologist, will highlight his lab's work on using
publicly-available molecular measurements to find new uses for drugs
including drug repositioning, discovering new treatable inflammatory
mechanisms of disease in type 2 diabetes, the evaluation of patients
presenting with whole genomes sequenced, and new work on integrating and
reusing the data that results from clinical trials.  Dr. Butte will
especially cover big data in biomedicine as a platform for innovation and
entrepreneurship.

<tr><td height="325" valign="bottom"><h1><a name="Collins"><img src="keynotes/Collins.jpg" width="335"></a>
<td><b>J.J. Collins</b><br>
Howard Hughes Medical Institute; Dept. of Biomedical Engineering & Center of Synthetic Biology, Boston University; Wyss Institute for Bioologically Inspired Engineering, Harvard University<br>
<a href="http://www.bu.edu/abl">Homepage</a></h1>
</tr>
<tr><td colspan=2><h4>Synthetic Biology and Biocomputation: Life Redesigned</h4>
Synthetic biology is bringing together engineers, computational
scientists and biologists to model, design and construct biological
circuits out of proteins, genes and other bits of DNA, and to use
these circuits to rewire and reprogram organisms.  These re-engineered
organisms are going to change our lives in the coming years, leading
to cheaper drugs, rapid diagnostic tests, and targeted therapies to
attack "superbugs". In this talk, we highlight recent efforts to model
and create synthetic gene networks and programmable cells, and discuss
a variety of synthetic biology applications in biocomputing,
biotechnology and biomedicine. 

<tr><td height="575" valign="bottom"><h1><a name="Ideker"><img src="keynotes/Ideker.jpeg" width="335"></a>
<td><b>Trey Ideker</b><br>
Professor of Medicine, University of California, San Diego<br>
<a href="http://idekerlab.ucsd.edu/">Homepage</a></h1>
</tr>
<tr><td colspan=2><h4>Network-based stratification of tumor mutations</h4>
Many forms of cancer have multiple subtypes with different causes and clinical outcomes. Somatic tumor genome sequences provide a rich new source of data for uncovering these subtypes but have proven difficult to compare, as two tumors rarely share the same mutations. Here we introduce network-based stratification (NBS), a method to integrate somatic tumor genomes with gene networks. This approach allows for stratification of cancer into informative subtypes by clustering together patients with mutations in similar network regions. We demonstrate NBS in ovarian, uterine and lung cancer cohorts from The Cancer Genome Atlas. For each tissue, NBS identifies subtypes that are predictive of clinical outcomes such as patient survival, response to therapy or tumor histology. We identify network regions characteristic of each subtype and show how mutation-derived subtypes can be used to train an mRNA expression signature, which provides similar information in the absence of DNA sequence.


<tr><td height="300" valign="bottom"><h1><a name="Mitchell"><img src="keynotes/Mitchell.jpg" width="335"></a>
<td><b>Tom Mitchell</b><br>
Fredkin University Professor, Department Head, Machine Learning Department, Carnegie Mellon University<br>
<a href="http://www.cs.cmu.edu/~tom/">Homepage</a></h1>
</tr>
<tr><td colspan=2><h4>How the Brain Represents the Meaning of Language</h4>
How does the human brain use neural activity to represent meanings of words, phrases, sentences and stories?  One way to study this question is to scan people's brains while they read.  We have been doing such experiments with fMRI (1 mm spatial resolution) and MEG (1 msec time resolution) brain imaging, then using machine learning methods to decode mental states from observed brain activity.  As a result, we have learned answers to questions such as "Are the neural encodings of word meaning the same in your brain and mine?", "What sequence of neurally encoded information flows through the brain during the half-second in which the brain comprehends a word?", and "What happens as the brain composes the meaning of multiple word phrases?"  This talk will summarize some of what we have learned, and newer questions we are currently working on.  We will also consider how new developments in machine learning may accelerate progress in data-intensive experimental sciences.

<tr><td height="600" valign="bottom"><h1><a name="Tishkoff"><img src="keynotes/Tishkoff.jpg" width="335"></a>
<td><b>Sarah A. Tishkoff</b><br>
David and Lyn Silfen University Professor, Depts. of Genetics and Biology, University of Pennsylvania<br>
<a href="http://www.med.upenn.edu/tishkoff/Lab/Tishkoff/Tishkoff.html">Homepage</a></h1>
</tr><td colspan=2><h4>African Evolutionary Genomics: Implications for human origins and adaptation</h4>
Africa is thought to be the ancestral homeland of all modern human
populations.  It is also a region of tremendous cultural, linguistic,
climatic, and genetic diversity.   Despite the important role that African
populations have played in human history, they remain one of the most
underrepresented groups in human genomics studies. A comprehensive
knowledge of patterns of variation in African genomes is critical for a
deeper understanding of human genomic diversity, the identification of
functionally important genetic variation, the genetic basis of adaptation
to diverse environments and diets, and the origins of modern humans.
Furthermore, a deeper understanding of African genomic variation will
provide the necessary foundation for powerful and efficient genome-wide
association and systems biology studies to identify coding and regulatory
variants that play a role in phenotypic variation including disease
susceptibility.   Comparison of African populations with similar genetic
ancestry but living in distinct environments may help distinguish both
genetic and environmental factors that contribute to variability in traits
of relevance both for health and disease.  We have used whole genome SNP
genotyping and high coverage sequencing analyses to characterize patterns
of genomic variation, ancestry, and local adaptation across ethnically and
geographically diverse African populations.   We have identified candidate
loci that play a role in adaptation to infectious disease, diet and high
altitude, as well as the short stature trait in African Pygmies.
Additionally, our studies shed light on human evolutionary history and
African population history.
</tr>
