
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" >

  <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:700|Pontano+Sans' rel='stylesheet' type='text/css'>
  <link rel="stylesheet" type="text/css" href="course.css">
  <title>RECOMB 2014 - April 2-5, Pittsburgh, Pennsylvania</title>
</head>
<body>

<!-- ========================================================================= -->
<header id="masthead">
<img src="748x205_level_header_city2.JPG" width="935" height="256">
</header>

<!-- ========================================================================= -->
<aside id="sidebar">
    <section>
      <nav id="global">
       <ul>
          <li><a href="index.html">Home</li>
          <li><a href="geninfo.html">General Information</a></li>
          <li><a href="http://en.wikipedia.org/wiki/Alumni_Hall_%28University_of_Pittsburgh%29">Conference Venue</a></li>
          <li><a href="program.html">Program</a></li>
          <li><a href="registration.html">Registration</a></li>
          <li><a href="travel.html">Travel & Visa</a></li>
  <li><a href="lodging.html">Accommodations</a></li>
          <li><a href="travelfellowships.html">Travel Fellowships</a></li>
          <li><a href="keynotes.html">Keynote Speakers</a></li>
          <li><a href="acceptedpapers.html">Accepted Papers</a></li>
          <li><a href="posters.html">Poster Sessions</a></li>
<li><a href="keydates.html">Key Dates</a></li>
          <li><a href="satellites.html">Satellite Workshops</a></li>
          <li><a href="callforpapers.html">Call for Papers</a></li>
          <li><a href="callforposters.html">Call for Posters</a></li>
          <li><a href="callforhighlights.html">Call for Highlights</a></li>
          <li><a href="submission.html">Submission</a></li>
          <li><a href="about.html">About RECOMB</a></li>
          <li><a href="http://www.cmu.edu/ua/pittsburgh/index.html">About Pittsburgh</a></li>
          <li><a href="committees.html">Committees</a></li>
          <li><a href="sponsorship.html">Sponsorship Opportunities</a>
<li><a href="sponsors.html">Sponsors</a></li>
          <li><a href="contact.html">Contact</a></li>
        </ul>
      </nav>
    </section>
</aside>

<div id="content">

<!-- ========================================================================= -->



<div id="content">

<!-- ========================================================================= -->
<section>
    <h1>18th Annual International Conference on Research in Computational Molecular Biology<br>
Pittsburgh, Pennsylvania<br>
April 2-5, 2014<p></h1>

<h1>Poster Sessions</h1>

<a href="posters.html#posters1">ISCB Poster Session</a> <a href="posters.html#posters2">NSF Poster Session</a>

<div id="content">

<!-- ========================================================================= -->
<section>
<p>
<a name="posters1"><h2>ISCB Poster Session<br>Thursday, April 3<br>
2:10pm-2:30pm, Ballroom, William Pitt Union</h2>
</a>

<p><b>1</b>	Progressive promoter element combinations classify conserved orthogonal plant circadian gene expression modules<br><i>
        Sandra Smieszek and Bart Przychodzen</i>
	
<p><b>3</b>	Folding Prediction of Bi-dimensional (2D) Structure of Proteins using an Ant Colony 
Optimization Algorithm (ACOA)<br><i>
John Fredy G&oacute;mez Rojas,  &Aacute;lvaro Andr&eacute;s Obreg&oacute;n Carre&ntilde;no, Dario Jos&eacute; Delgado Quintero and Rodrigo Torres</i>

	
<p><b>5</b>	Exploring distance between fusion genes in human genetic interaction network <br><i>
          Nanfang Xu</i>
	
<p><b>7</b>	Identification of Gene Biomarkers for Anti-Cancer Drugs Using Parallel PLS-Based Models <br><i>
          Iraj Hosseini, R. Joseph Bender and Feilim Mac Gabhann</i>
	
<p><b>9</b>	eFINDSITEPPI: Structure Based Prediction of Protein-Protein Interactions<br><i>
          Surabhi Maheshwari and Michal Brylinski</i>
	
<p><b>11</b>	Source-target connectivity in networks of uncertain topology<br><i>
          Andrei Todor, Haitham Gabr, Alin Dobra and Tamer Kahveci</i>
	
<p><b>13</b>	Validated methods for predictive genetics<br><i>
          Michael R. Brent, Ezekial Maier, Brian Haynes and Tamara Doering</i>
	
<p><b>15</b>	Terrace aware phylogenomic inference<br><i>
          Olga Chernomor, Bui Quang Minh and Arndt von Haeseler</i>

	
<p><b>17</b>	Deciphering the chromatin-related signaling from histone modification and chromatin modifier data<br><i>
          Juliane Perner, Julia Lasserre, Martin Vingron and Ho-Ryun Chung</i>
	

<p><b>19</b>	Computational analysis of protein-ligand interactions of enzyme genes involved in sulphate reduction<br><i>
          Paul Leyva-Miranda, Fernando Bastida-Gonzalez, Gemma Ramirez-Salinas, 
          Jose Correa-Basurto, Paola Zarate-Segura and Claudia Guerrero-Barajas</i>
	
<p><b>21</b>	Identification of Conserved RNA Structural Motifs using A Subgraph Random Sampling Approach<br><i>
          Jiajie Huang, Kejie Li and Michael Gribskov</i>
	
<p><b>23</b>	A Global Profiling of Alternative Splicing Exon Expression Patterns across Multiple Human 
Tissues with RNA-Seq Data<br><i>
          Zhiyi Qin, Yi Xing and Xuegong Zhang</i>
	
<p><b>25</b>	The Prediction of Protein-Protein Interactions between Myoglobin and Haptoglobin Through Computational Modeling and Confirmation of Binding through Mass Spectrometry<br><i>
          Ololade Fatunmbi, Rinat Abzalimov, Anne Gershenson and Igor Kaltashov</i>
	
<p><b>27</b>	Finding Differential Splice Junctions as Biomarkers in Prostate Cancer from Next Generation Sequencing Data<br><i>
          Iman Rezaeian, Ahmad Tavakoli, Dora Cavallo-Medved, Lisa Porter 
          and Luis Rueda</i>
	
<p><b>29</b>	Using a Hierarchical Classification Model for Predicting Ten Breast Cancer Subtypes<br><i>
          Forough Firoozbakht, Iman Rezaeian, Alioune Ngom, Lisa Porter and Luis Rueda</i>

	
<p><b>31</b>	An optimal experimental design scheme for learning Boolean network models<br><i>
          Nir Atias, Michal Gershenzon, Katia Labazin and Roded Sharan</i>

<p><b>33</b>	A differential co-expression networks approach to uncovering mechanisms underlying skeletal muscle pathophysiology<br><i>
          Kavitha Mukund and Shankar Subramaniam</i>
	
<p><b>35</b>	Rewiring of airway gene expression-based networks by bronchial dysplasia 
and lung cancer<br><i>
          Anna Tassinari, Bin Zhang, Duncan Whitney, Kate Porta, Stephen Lam, 
          Marc Lenburg, Avrum Spira and Jennifer Beane</i>
	
<p><b>37</b>	Finding network motifs using MCMC sampling<br><i>
          Tanay Kumar Saha and Mohammad Al Hasan</i>
	
<p><b>39</b>	Improvements in protein function prediction using confidence in protein interactions<br><i>
          Kathryn Doroschak, Thomas Schaffner and Lenore Cowen</i>
	
<p><b>41</b>	BisKit: A pipeline for large-scale bisulfite sequencing analysis<br><i>
          Jawon Song, Nathan M Springer and Matthew W Vaughn</i>
	
<p><b>43</b>	A novel method for identifying OR relationship in eQTL mapping<br><i>
          Zhiyong Wang, Jinbo Xu and Xinghua Shi</i>
	
<p><b>45</b>	Higher-order chromatin domains link eQTLs with the expression of far-away genes<br><i>
          Geet Duggal, Hao Wang and Carl Kingsford</i>

	
<p><b>47</b>	Dissecting Germ Cell Metabolism through Network Modeling 
<br><i>
         Leanne Whitmore and Ping Ye </i>

	
<p><b>49</b>	Hybrid Approach Using SVM and MM2 in Splice Site Junction Identification<br><i>
          Srabanti Bhunia and Deepak Garg</i>
	
<p><b>51</b>	Biomedical Data Fusion by Simultaneous Matrix Tri-factorization<br><i>
          Marinka Zitnik and Blaz Zupan</i>
	
<p><b>53</b>	High-throughput genome scaffolding from in vivo DNA interaction frequency<br><i>
          Noam Kaplan and Job Dekker</i>
	
<p><b>55</b>	Sex-specific expression of hepatic lincRNAs: potential role in growth hormone (GH) regulation of liver epigenetic states.<br><i>
          Tisha Melia and David Waxman</i>
	
<p><b>57</b>	Algorithms and complexity results for genome mapping problems<br><i>
          Cedric Chauve, Jan Manuch, Ashok Rajaraman and Joao Paulo Pereira Zanetti</i>
	
<p><b>59</b>	Privacy Preserving Calculation of Fisher Criterion Score for Informative Gene Selection<br><i>
          Suxin Guo, Sheng Zhong and Aidong Zhang</i>
	
<p><b>61</b>	Efficient Alignment and Variant Calling at Scale Using the Cloud<br><i>
          Frank A Nothaft, Matt Massie, Ameet Talwalkar, Bill Bolosky, Jeremy Elson 
          and David Patterson</i>

	
<p><b>63</b>	Computational Analysis of Domain Interactions Between Components 
of the SCF Ligase<br><i>
          Mina Maleki, Mohammad Haj Dezfulian, William Crosby and Luis Rueda</i>
	
<p><b>65</b>	Reconstructing dynamic microRNA-regulated interaction networks<br><i>
          Marcel Schulz, Kusum Pandit, Christian Lacks Lino Cardenas, 
          Namasivayam Ambalavanan, Naftali Kaminski and Ziv Bar-Joseph</i>
	
<p><b>67</b>	Active Learning Based Matrix Completion for Predicting Drug-Target Interactions<br><i>
          Maja Temerinac-Ott, Armaghan W. Naik and Robert F. Murphy</i>
	
<p><b>69</b>	Assessing Collective Disease Association of Multiple Genomic Loci<br><i>
          Marzieh Ayati, Pamela L. Clark, Mark R. Chance and Mehmet Koyuturk</i>
	
<p><b>71</b>	A Combinatorial Perspective of the Protein Inference Problem<br><i>
          Chao Yang, Zengyou He and Weichuan Yu</i>
	
<p><b>73</b>	Computational modeling of miRNA biogenesis: an insight into transcription 
and processing<br><i>
          Annalisa Marsico, Thomas Conrad, Martin Vingron and Ulf Ørom</i>
	
<p><b>75</b>	Identification of Gene Biomarkers for Anti-Cancer Drugs Using Parallel PLS-Based Models<br><i>
          Iraj Hosseini, R. Joseph Bender and Feilim Mac Gabhann</i>
	
<p><b>77</b>	Generative models of the spatial organization of neurons <br><i>
          Rebecca Elyanow, Xuexia Jiang, Devin Sullivan and Robert F. Murphy </i>
	
<p><b>79</b>	A combinatorial approach for analyzing intra-tumor heterogeneity from high-throughput 
sequencing data <br><i>
          Iman Hajirasouliha, Ahmad Mahmoody and Ben Raphael</i>
	
<p><b>81</b>	Transfer learning for reconstructing dynamic signaling and regulatory networks  <br><i>      
          Siddhartha Jain and Ziv Bar-Joseph</i>
	
<p><b>83</b>	Fast Lossless Compression via Cascading Bloom Filters <br><i>
          Roye Rozov, Ron Shamir and Eran Halperin</i>
	
<p><b>85</b>	Loads-To-Nodes: A Framework for Managing Biological Databases on Massively Parallel 
Environments <br><i>
          Dana Silverbush and Roded Sharan</i>
	
<p><b>87</b>	Network orientation via shortest paths<br><i>
          Ahmad Salah and Li Kenli</i>
	
<p><b>89</b>	Molecular function profile meta-analysis deciphers common molecular and pathological 
signatures of leukemia stem cells <br><i>
          Xinan Yang, Aurelie Desgardin, Jianjun Chen, Jill de Jong and John Cunningham</i>
	
<p><b>91</b>	microRNAs located in the Hox gene clusters are implicated in Huntington.s disease pathogenesis <br><i>
          Vinay Kartha, Andrew Hoss, Xianjun Dong, Jeanne Latourelle, 
          Alexandra Dumitriu, Tiffany Hadzi, Marcy MacDonald, James Gusella, 
          Schahram Akbarian, Jiang-Fan Chen, Zhiping Weng and Richard Myers</i>
	
<p><b>93</b>	A Neutrophil Phenotype Model for Extracorporeal Treatment of Sepsis <br><i>
          Robert Sheehan, Alex Malkin, Shibin Mathew, Heinz Redl and Gilles Clermont</i>
	
<p><b>95</b>	TEAM: TE Annotation from Methylation <br><i>
          Gregory Zynda</i>
	
<p><b>97</b>	Discovering conserved secondary structures in lncRNAs <br><i>
          Ardalan Naseri and Shaojie Zhang</i>
	
<p><b>99</b>	MRFalign: Protein Homology Detection through Alignment of Markov Random Fields   <br><i>
          Jianzhu Ma, Sheng Wang, Zhiyong Wang and Jinbo Xu	</i>



<p>
<h2>
<a name="posters2">NSF Poster Session<br>Friday, April 4<br>
5:30pm-7:00pm, Ballroom, William Pitt Union</h2></a>

<p><b>2</b>	SynDeC: High-throughput synapse detection and characterization in electron microscopy images <br><i>
          Kelvin Liu, Saket Navlakha, Santosh Chandrasekaran, Alison L. Barth and Ziv Bar-Joseph</i>
	
<p><b>4</b>	TACO: a general-purpose tool for predicting cell-type.specific transcription factor dimers <br><i>
          Aleksander Jankowski, Shyam Prabhakar and Jerzy Tiuryn</i>
	
<p><b>6</b>	Speeding Up Sequence Alignment for MetaGenomic Clustering Using Early Stopping <br><i>
          Louis Oliphant, Nicola Pedretti and Matthew Pozderac</i>
	
<p><b>8</b>	Efficient Modeling and Active Learning of Biological Responses: Learning without Prior Knowledge <br><i>
          Armaghan Naik, Joshua Kangas, Devin Sullivan, Christopher Langmead and Robert F. Murphy</i>
	
<p><b>10</b>	Convex Risk Minimization To Infer Networks From Probabilistic Diffusion Data At Multiple Scales <br><i>
          Emre Sefer and Carl Kingsford</i>
	
<p><b>12</b>	Effects of Geometry on Computational Kinetics Models of Early Endosomes <br><i>
          Rohan Arepally, Devin Sullivan and Robert F. Murphy</i>
	
<p><b>14</b>	omiRas: a Web server for differential expression analysis of miRNAs derived from small RNA-Seq data <br><i>
          Soeren Mueller, Lukas Rycak, Peter Winter, GüKahl and BjöRotter</i>
	
<p><b>16</b>	Variant detection model with improved robustness and accuracy for low-depth targeted next-generation sequencing data <br><i>
          Patrick Flaherty and Fan Zhang</i>
	
<p><b>18</b>	A new method for gene-fusion detection: Fusion-Detection by Assembly (FDA)<br><i>
          Nanfang Xu</i>
	
<p><b>20</b>	Algorithms for inferring evolutionary models of tumor progression from single-cell heterogeneity data <br><i>
          Salim Akhter Chowdhury, Alejandro Schaffer, Stanley Shackney, 
          Darawalee Wangsa, Kerstin Heselmeyer-Haddad, Thomas Ried 
          and Russell Schwartz</i>
	
<p><b>22</b>	Identification of Binding Site Enrichment of Ikaros in Leukemia with a new data analysis pipeline <br><i>
          Xiaokang Pan, Chunhua Song, Sadie Steffens and Sinisa Dovat</i>
	
<p><b>24</b>	An Image-derived 3D Generative Model for the Spatial Dependence of Multiple Protein Patterns Reveals Differences between Plant Protoplasts in Different Drug Conditions <br><i>   
          Pingshan Zhang, Gregory Johnson, Joshua Kangas, Alexander Dovzhenko, 
          Klaus Palme and Robert F. Murphy</i>

<p><b>26</b>	Mathematical modeling of Ras/MAPK signaling network kinetics provides insights into mechanisms of FGF2-induced cell cycle arrest in the K-Ras-driven mouse Y1 adrenocortical tumor cells <br><i>
          Marcelo S. Reis, Matheus H.S. Dias, Fáo Nakano, Junior Barrera and Hugo A. Armelin</i>
	
<p><b>28</b>	Epistatic interactions during somatic evolution in cancer <br><i>
          Navodit Misra</i>


<p><b>30</b>	CeTICSdb: a platform for interdisciplinary research that allows quantitative and qualitative -omics analyses and mathematical modeling of signaling networks <br><i>
          Marcelo S. Reis, Milton Y. Nishiyama-Jr, Daniel F. Silva, Ináo L.M. Junqueira-De-Azevedo, Julia P.C. Da Cunha, Junior Barrera, Leo K. Iwai, Solange M.T. Serrano and Hugo A. Armelin</i>
	
<p><b>32</b>	IQ-TREE: A simple and efficient stochastic search algorithm for estimating maximum-likelihood phylogenies <br><i>
          Lam Tung Nguyen, Heiko Schmidt, Arndt von Haeseler and Bui Quang Minh</i>
	
<p><b>34</b>	Computing Free Energies With PyBrella <br><i>
          Charles Yuan and David Koes</i>
	
<p><b>36</b>	Accelerating systems biology computation: Rapid estimation of equilibrium and kinetic quantities via weighted ensemble sampling <br><i>
          Rory Donovan, Daniel Zuckerman, James Faeder, Andrew Sedgewick, Jun Ma 
          and Markus Dittrich</i>
	
<p><b>38</b>	Kraken: ultrafast metagenomic sequence classification using exact alignments <br><i>
          Derrick Wood and Steven Salzberg</i>
	
<p><b>40</b>	Cloud Computing Chemogenomics Knowledgebases and Cheminformatics Tools for Polypharmacology Networks<br><i>
          Haibin Liu, Lirong Wang and Xiang-Qun Xie</i>
	
<p><b>42</b>	Ligand Affinity Prediction with Grid Inhomogeneous Solvation Theory and smina <br><i>
          James Castiglione and David Koes</i>

	
<p><b>44</b>	Probabilistic Method for Detecting Copy Number Variation in a Fetal Genome using Maternal Plasma Sequencing <br><i>
          Ladislav Rampasek, Aryan Arbabi and Michael Brudno</i>
	
<p><b>46</b>	ENNET: A Method for Gene Regulatory Network Inference <br><i>
          Janusz Slawek and Tomasz Arodz</i>
	
<p><b>48</b>	MASSIVELY PARALLEL PARE PREDICTION TOOL (MPPP): High-performance 
identification of plant sRNA targets at genome level <br><i>
          Atul Kakrana, Reza Hammond, Parth Patel, Mayumi Nakano and Blake Meyers</i>
	
<p><b>50</b>	Pluribus - Using Multiple Manifestations of a Sequencing Error in a Suffix-Tree to Improve Correction <br><i>
          Daniel Savel, Thomas Laframboise, Ananth Grama and Mehmet Koyuturk</i>
	
<p><b>52</b>	Identification of Alternative Topological Domains in Chromatin <br><i>
          Darya Filippova, Rob Patro, Geet Duggal and Carl Kingsford</i>
	
<p><b>54</b>	Benchmarking of base pair interaction predictions from RNA 3D structures <br><i>
          Shahidul Islam and Shaojie Zhang</i>
	
<p><b>56</b>	Identification of Cohesive and Reproducible Patterns of Epistasis in Psoriasis <br><i>
          Pam Clark, Mark R. Chance and Mehmet Koyuturk	</i>

<p><b>58</b>	Comparative Assembly Hubs: Web Accessible Browsers for Comparative Genomics <br><i>
          Ngan Nguyen, Glenn Hickey, Brian Raney, Joel Armstrong, Hiram Clawson, Ann Zweig, Jim Kent, David Haussler and Benedict Paten</i>

<p><b>60</b>	Effects of EPHB6 Expression in Human Breast Tumours on Metastasis and Survival <br><i>
          Matthew Shannon, Andrew Freywald and Anthony Kusalik</i>

<p><b>62</b>	Novel transcriptional signatures of host immune response can distinguish viral hemorrhagic fever infection from respiratory virus infection <br><i>
          Sara Garamszegi</i>

<p><b>64</b>	Examination of Stability of HMMs When Applied to CpG Island Prediction <br><i>
          Daniel Hogan and Anthony Kusalik</i>

<p><b>66</b>	Construction of a Metabolic Interactome for Insight into Microbial Ecosystems <br><i>
          Brian Granger</i>

<p><b>68</b>	GLAD: A mixed-membership model for heterogeneous tumor subtype classification  <br><i>     
          Patrick Flaherty, Jon McAuliffe and Hachem Saddiki</i>

<p><b>70</b>	Integration of Disparate Mapping Algorithms for Improved Alignment of Short Sequencing Reads <br><i>
          Matthew Ruffalo, Thomas Laframboise and Mehmet Koyuturk</i>

<p><b>72</b>	Reference-free Inference of Tumor Phylogenies from Single-Cell Sequencing Data <br><i>
          Ayshwarya Subramanian and Russell Schwartz</i>

<p><b>74</b>	Predictive Models for Head and Neck Cancer Stage from Genomic and Proteomic Data <br><i>
          Chanchala D. Kaddi and May D. Wang</i>

<p><b>76</b>	Estimating insertion, deletion, and recombination rates of Insertion Sequence (IS) elements in the genome of Escherichia coli <br><i>
          Heewook Lee, Ellen Popodi, Patricia Foster and Haixu Tang</i>

<p><b>78</b>	Tumor heterogeneity in Chronic Lymphocytic Leukemia <br><i>
          Luis Zapata and Stephan Ossowski</i>

<p><b>80</b>	Pareto-optimality and Consensus Support in Duplication-Transfer-Loss Reconciliation <br><i>
          Ran Libeskind-Hadas, Yi-Chieh Wu and Mukul Bansal</i>

<p><b>82</b>	BlindCall: ultra-fast base-calling of high-throughput sequencing data by blind deconvolution <br><i>
          Chengxi Ye, Chiaowen Hsiao and Hector Corrada Bravo</i>

<p><b>84</b>	Benchmark Analysis of Algorithms for Reconstructing Full Splice Forms from RNA-Seq <br><i>
          Katharina Hayer, Angel Pizarro, John Hogenesch and Gregory Grant</i>

<p><b>86</b>	A method for predicting protein-protein interaction types <br><i>
          Yael Silberberg, Martin Kupiec and Roded Sharan</i>

<p><b>88</b>	Extensive Cross-regulation in Post-transcriptional Regulatory Network in Drosophilia <br><i>
          Marcus Stoiber, Gemma May, Michael Duff, Robert Obar, Spyros Artavanis-Tsakonas, Ben Brown, Brenton Graveley and Susan Celniker</i>

<p><b>90</b>	De novo discovery of transcription factor footprints from high-resolution sequencing data<br><i>
          Alessandro Mammana and Ho-Ryun Chung</i>

<p><b>92</b>	DREISS: Decomposition of gene Regulatory network into External and Internal components based on State Space models<br><i>
          Daifeng Wang, Fei He, Sergei Maslov and Mark Gerstein</i>


<p><b>94</b>	ReconModels.org: An Online Database of Reconstructed Metabolic Networks<br><i>
          A. Ercument Cicek, Bryan Marty, Xinjian Qi and Gultekin Ozsoyoglu</i>

<p><b>96</b>	Discovering conserved secondary structures in lncRNAs<br><i>
          Ardalan Naseri and Shaojie Zhang</i>

<p><b>98</b>	RVD2: An ultra-sensitive variant detection model for low-depth targeted next-generation sequencing data <br><i>
          Patrick Flaherty and Yuting He</i>



