
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" >

  <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:700|Pontano+Sans' rel='stylesheet' type='text/css'>
  <link rel="stylesheet" type="text/css" href="course.css">
  <title>RECOMB 2014 - April 2-5, Pittsburgh, Pennsylvania</title>
</head>
<body>

<!-- ========================================================================= -->
<header id="masthead">
<img src="748x205_level_header_city2.JPG" width="935" height="256">
</header>

<!-- ========================================================================= -->
<aside id="sidebar">
    <section>
      <nav id="global">
       <ul>
          <li><a href="index.html">Home</li>
          <li><a href="geninfo.html">General Information</a></li>
          <li><a href="http://en.wikipedia.org/wiki/Alumni_Hall_%28University_of_Pittsburgh%29">Conference Venue</a></li>
          <li><a href="program.html">Program</a></li>
          <li><a href="registration.html">Registration</a></li>
          <li><a href="travel.html">Travel & Visa</a></li>
  <li><a href="lodging.html">Accommodations</a></li>
          <li><a href="travelfellowships.html">Travel Fellowships</a></li>
          <li><a href="keynotes.html">Keynote Speakers</a></li>
          <li><a href="acceptedpapers.html">Accepted Papers</a></li>
          <li><a href="posters.html">Poster Sessions</a></li>
<li><a href="keydates.html">Key Dates</a></li>
          <li><a href="satellites.html">Satellite Workshops</a></li>
          <li><a href="callforpapers.html">Call for Papers</a></li>
          <li><a href="callforposters.html">Call for Posters</a></li>
          <li><a href="callforhighlights.html">Call for Highlights</a></li>
          <li><a href="submission.html">Submission</a></li>
          <li><a href="about.html">About RECOMB</a></li>
          <li><a href="http://www.cmu.edu/ua/pittsburgh/index.html">About Pittsburgh</a></li>
          <li><a href="committees.html">Committees</a></li>
          <li><a href="sponsorship.html">Sponsorship Opportunities</a>
<li><a href="sponsors.html">Sponsors</a></li>
          <li><a href="contact.html">Contact</a></li>
        </ul>
      </nav>
    </section>
</aside>

<div id="content">

<!-- ========================================================================= -->



<div id="content">

<!-- ========================================================================= -->
<section>
    <h1>18th Annual International Conference on Research in Computational Molecular Biology<br>
Pittsburgh, Pennsylvania<br>
April 2-5, 2014<p></h1>

<h1>RECOMB 2014 PROGRAM</h1>

<h2>
The conference proceedings <a href="http://link.springer.com/openurl.asp?genre=issue&issn=0302-9743&volume=8394">(<i>Lecture Notes in Computer Science</i> Volume 8394, 2014)</a> are now online. Attendees will receive printed and electronic copies of the proceedings.
</h2>
<p>
<h1><a href="program.html#April2">April 2</a>, <a href="program.html#April3">April 3</a>, <a href="program.html#April4">April 4</a>, <a href="program.html#April5">April 5</a>.</h1>
<table class="schedule">
<tr><td colspan=2><h2><a name="April1">Tuesday, April 1 (Pittsburgh Athletic Association)</a></h2>

<tr><td>18:00-21:00<td>Opening Night Reception

<tr><td colspan=2><a name="April2"><h2>Wednesday, April 2 (Alumni Hall)</a> </h2>

<tr><td>8:45-9:00<td>Opening Remarks and Welcome by Subra Suresh, President, Carnegie Mellon University

<tr><td>9:00-10:40	<td ><b>Session 1. Chair: Russell Schwartz, Carnegie Mellon University</b>
 	 
<tr><td>9:00-10:00	<td><b>KEYNOTE: James Collins. <br><i><a href="keynotes.html#Collins">Synthetic Biology and Biocomputation: Life Redesigned</a></i></b>
 	 
<tr><td>10:00-10:20	<td>Armin T&ouml;pfer, Tobias Marschall, Rowena Bull, Fabio Luciani, Alexander Sch&ouml;nhuth and Niko Beerenwinkel. <br><i>Viral quasispecies assembly via maximal clique enumerationion</i>
 	 
<tr><td>10:20-10:40	<td>Henry C.M. Leung, S.M. Yiu and Francis Chin. <br><i>IDBA-MTP: A Hybrid MetaTranscriptomic Assembler Based on Protein Information</i>

<tr><td>10:40-11:10 <td><b>BREAK</b>

<tr><td>11:10-12:10	<td><b>Session 2. Chair: Michael Waterman, University of Southern California</b>
 	 
<tr><td>11:10-11:30	<td>Rayan Chikhi, Antoine Limasset, Shaun Jackman and Jared Simpson and Paul Medvedev. <br><i>On the representation of de Bruijn graphs</i>
 	 
<tr><td>11:30-11:50	<td>Yana Safonova, Anton Bankevich and Pavel Pevzner. <br><i>DipSPAdes: Assembler for Highly Polymorphic Diploid Genomes</i>
 	 
<tr><td>11:50-12:10	<td><b>HIGHLIGHT:</b> Noam Kaplan and Job Dekker. <br><i>High-throughput genome scaffolding from in vivo DNA interaction frequency</i>
 	 
<tr><td>12:10-13:30	<td>LUNCH

<tr><td>13:30-15:10	<td><b>Session 3. Chair: Ron Shamir, Tel Aviv University</b>
 	 
<tr><td>13:30-13:50	<td>Jianzhu Ma, Sheng Wang and Jinbo Xu. <br><i>MRFalign: Protein Homology Detection through Alignment of Markov Random Fields</i>
 	 
<tr><td>13:50-14:10	<td>Siavash Mirarab, Nam-Phuong Nguyen and Tandy Warnow. <br><i>PASTA: ultra-large multiple sequence alignment</i>
 	 
<tr><td>14:10-15:10	<td><b>PANEL: Publication policies in RECOMB</b>   <br>Moderator: Ron Shamir, Tel Aviv University       <br>Confirmed Participants:<br> Irit Gat-Viks, Tel Aviv University<br>Carl Kingsford, Carnegie Mellon University<br>Teresa Przytycka, NIH <br>Martin Vingron, MPI Berlin               
 	                        
<tr><td>15:10-15:40	<td><b>BREAK</b>

<tr><td>15:40-17:00	<td><b>Session 4. Chair: Tandy Warnow, University of Texas, Austin</b>
 	 
<tr><td>15:40-16:00	<td>Mingfu Shao, Yu Lin and Bernard Moret. <br><i>An Exact Algorithm to Compute the DCJ Distance for Genomes with Duplicate Genes</i>
 	 
<tr><td>16:00-16:20	<td>Yu Zheng and Louxin Zhang. <br><i>Reconciliation with Non-binary Gene Trees Revisited</i>
 	 
<tr><td>16:20-16:40	<td>Ngan Nguyen, Glenn Hickey, Daniel Zerbino, Brian Raney, Dent Earl, Joel Armstrong, David Haussler and Benedict Paten. <br><i>Building a Pangenome Reference for a Population</i>
 	 
<tr><td>16:40-17:00	<td><b>HIGHLIGHT:</b> John A. Capra, Maureen Stolzer, Dannie Durand and Katherine S. Pollard. <br><i>How old is my gene?</i>

<tr><td>17:00	<td>Free afternoon

<tr><td colspan=2><h2><a name="April3">Thursday, April 3 (William Pitt Union)</a></h2>


<tr><td>9:00-10:00	<td><b>Session 5. Chair: Sorin Istail, Brown University</b>
 	
<tr><td>9:00-9:20	<td>Emily Berger, Deniz Yorukoglu and Bonnie Berger. <br><i>HapTree: A novel Bayesian framework for single individual polyplotyping using NGS data</i>
 	 
<tr><td>9:20-9:40	<td>Murray Patterson, Tobias Marschall, Nadia Pisanti, Leo van Iersel, Leen Stougie, Gunnar Klau and Alexander Sch&ouml;nhuth. <br><i>WhatsHap: Haplotype Assembly for Future-Generation Sequencing Reads</i>
 	 
<tr><td>9:40-10:00	<td><b>HIGHLIGHT:</b> Mohamed Aziz Mezlini, Eric J. M. Smith, Marc Fiume, Orion Buske, Gleb L. Savich, Sohrab Shah, Sam Aparicio, Derek Chiang, Anna Goldenberg and Michael Brudno. <br><i>iReckon: Simultaneous isoform discovery and abundance estimation from RNA-seq data</i>
 	 
<tr><td>10:00-10:30	<td><b>BREAK</b>
 
<tr><td>10:30-12:10	<td><b>Session 6. Chair: Bonnie Berger, Massachusetts Institute of Technology</b>

<tr><td>10:30-11:30	<td><b>KEYNOTE: Ian Baldwin. <br><i><a href="keynotes.html#Baldwin">Using our biological legacy before it's lost; melding computational and natural history approaches to provide economic arguments for conservation</a></b></i>
 	 
<tr><td>11:30-11:50	<td>Rui Wang and Scott Schmidler. <br><i>Bayesian Multiple Protein Structure Alignment</i>

<tr><td>11:50-12:10	<td>Shutan Xu, Shuxue Zou and Lincong Wang. <br><i>A geometric clustering algorithm and its applications to structural data</i>
	
<tr><td>12:10-14:30	<td><b>LUNCH + <a href="http://www.compbio.cmu.edu/recomb/posters.html#posters1">ISCB POSTER SESSION</a></b>

<tr><td>14:30-16:10	<td><b>Session 7. Chair: Takis Benos, University of Pittsburgh</b>
 	 
<tr><td>14:30-15:30	<td><b>KEYNOTE: Atul Butte. <br><i><a href="keynotes.html#Butte">Translating a trillion points of data into therapies, diagnostics, and new insights into disease</a></i></b>
 	 

<tr><td>15:30-15:50	<td>Shay Zakov and Vineet Bafna. <br><i>Reconstructing Breakage Fusion Bridge architectures using noisy copy numbers</i>
 	 
<tr><td>15:50-16:10	<td>Arun Konagurthu, Parthan Kasarapu, Lloyd Allison, James Collier and Arthur Lesk. <br><i>On sufficient statistics of least-squares superposition of vector sets</i>
 	 
<tr><td>16:10-16:30	<td><b>BREAK</b>

<tr><td>16:30-17:30	<td><b>Session 8. Chair: S. Cenk Sahinalp, Simon Fraser University</b>

<tr><td>16:30-16:50	<td>Adam Bloniarz, Ameet Talwalkar, Jonathan Terhorst, Michael Jordan, David Patterson, Bin Yu and Yun Song. <br><i>Changepoint Analysis for Efficient Variant Calling</i>
 	 
<tr><td>16:50-17:10	<td>Y. William Yu, Deniz Yorukoglu and Bonnie Berger. <br><i>Traversing the k-mer landscape of NGS read datasets for quality score sparsification</i>
	
<tr><td>17:10-17:30	<td>Chen-Ping Fu, Vladimir Jojic and Leonard Mcmillan. <br><i>An Alignment-Free Regression Approach for Estimating Allele-Specific Expression using RNA-Seq Data</i>
 	 
	
<tr><td>18:00-22:00	<td><b>Conference Banquet - <a href="https://www.google.com/maps/place/4400+Forbes+Ave/@40.4435041,-79.9495881,17z/data=!3m1!4b1!4m2!3m1!1s0x8834f2265ce2f431:0x17c449be36d166a0">Carnegie Museums</a></b>



<tr><td colspan=2><h2><a name="April4">Friday, April 4 (William Pitt Union)</a></h2>

<tr><td>9:00-10:00	<td><b>Session 9. Chair: Ziv Bar-Joseph, Carnegie Mellon University</b>

<tr><td>9:00-9:20	<td>Arne Reimers, Frank Bruggeman, Brett Olivier and Leen Stougie. <br><i>Fast Flux Module Detection using Matroid Theory</i>
 	 
<tr><td>9:20-9:40	<td>Shaun Mahony, Matthew Edwards, Esteban Mazzoni, Richard Sherwood, Akshay Kakumanu, Carolyn Morrison, Hynek Wichterle and David Gifford. <br><i>An integrated model of multiple-condition ChIP-seq data reveals predeterminants of Cdx2 binding</i>
 	 
<tr><td>9:40-10:00	<td>Jianling Zhong, Todd Wasson and Alexander Hartemink. <br><i>Learning protein-DNA interaction landscapes by integrating experimental data through computational models</i>
 	 
<tr><td>10:00-10:30	<td><b>BREAK</b>

<tr><td>10:30-12:10	<td><b>Session 10. Chair: Carl Kingsford, Carnegie Mellon University</b>
 	 
<tr><td>10:30-11:30	<td><b>KEYNOTE: Tom Mitchell. <br><i><a href="keynotes.html#Mitchell">How the Brain Represents the Meaning of Language</a></b>
 	 
<tr><td>11:30-11:50	<td>Hetu Kamisetty, Bornika Ghosh, Christopher James Langmead and Chris Bailey-Kellogg. <br><i>Learning Sequence Determinants of Protein:protein Interaction Specificity with Sparse Graphical Models</i>
 	 
<tr><td>11:50-12:10	<td><b>HIGHLIGHT:</b> Irit Gat-Viks, Nicolas Chevrier, Roni Wilentzik, Thomas Eisenhaure, Raktima Raychowdhury, Yael Steuerman, Nir Hacohen, Ido Amit and Aviv Regev. <br><i>Deciphering molecular circuits from genetic variation underlying transcriptional responsiveness to stimuli</i>
 	 
<tr><td>12:10-13:30	<td><b>LUNCH</b>

<tr><td>13:30-15:10	<td><b>Session 11. Chair: Serafim Batzoglou, Stanford University</b>
 	 
<tr><td>13:30-14:30	<td><b>KEYNOTE: Trey Ideker. <br><i><a href="keynotes.html#Ideker">Network-based stratification of tumor mutations</b></i></a>
 	 
<tr><td>14:30-14:50	<td>Wen-Yun Yang, Farhad Hormozdiari, Eleazar Eskin and Bogdan Pasaniuc. <br><i>A Spatial-Aware Haplotype Copying Model with Applications to Genotype Imputation</i>
 	 
<tr><td>14:50-15:10	<td>Derek Aguiar, Eric Morrow and Sorin Istrail. <br><i>Tractatus: an exact and subquadratic algorithm for inferring identity-by-descent multi-shared haplotype tracts</i>
 	 
<tr><td>15:10-15:40	<td><b>BREAK </b>

<tr><td>15:40-17:00	<td><b>Session 12. Chair: Vineet Bafna, University of California, San Diego</b>
 	 
<tr><td>15:40-16:00	<td>Zhanyong Wang, Jae-Hoon Sul, Sagi Snir, Jose A. Lozano and Eleazar Eskin. <br><i>Gene-Gene Interactions Detection Using A Two-stage Model</i>
 	 
<tr><td>16:00-16:20	<td>Kelley Harris, Sara Sheehan, John Kamm and Yun S. Song. <br><i>Decoding Coalescent Hidden Markov Models in Linear Time</i>

<tr><td>16:20-16:40	<td>Adrian Guthals, Christina Boucher and Nuno Bandeira. <br><i>The generating function approach for peptide identification in spectral networks </i>

<tr><td>16:40-17:00	<td><b>HIGHLIGHT:</b> Joseph Paulson, O Colin Stine, Hector Corrada Bravo and Mihai Pop. <br><i>Differential abundance analysis for microbial marker-gene surveys </i>

<tr><td>17:00-17:30	<td>Business Meeting
 	 	 
<tr><td>17:30-19:00	<td><b>REFRESHMENTS + <a href="http://www.compbio.cmu.edu/recomb/posters.html#posters2">NSF POSTER SESSION</a></b>



<tr><td colspan=2><h2><a name="April5">Saturday, April 5 (William Pitt Union)</a></h2>

<tr><td>9:00-10:00	<td><b>Session 13. Chair: Martin Vingron, MPI Berlin</b>

<tr><td>9:00-9:20	<td>Raunak Shrestha, Ermin Hodzic, Jake Yeung, Kendric Wang, Thomas Sauerwald, Phuong Dao, Shawn Anderson, Himisha Beltran, Mark A. Rubin, Colin Collins, Gholamreza Haffari and S. Cenk Sahinalp. <br><i>HIT.nDRIVE: Multi-Driver Gene Prioritization based on Hitting Time</i>
 	 
<tr><td>9:20-9:40	<td>Jan Hoinka, Alexey Berezhnoy, Zuben E. Sauna, Eli Gilboa and Teresa Przytycka. <br><i>AptaCluster - A Method to Cluster HT-SELEX Aptamer Pools and Lessons from its Application</i>
 	 
<tr><td>9:40-10:00	<td><b>HIGHLIGHT:</b> Soheil Feizi, Daniel Marbach, Muriel Medard and Manolis Kellis. <br><i>Network deconvolution as a general method to distinguish direct dependencies in networks</i>
 	 
<tr><td>10:00-10:30	<td><b>BREAK</b>

<tr><td>10:30-12:10	<td><b>Session 14. Chair: Roded Sharan, Tel Aviv University</b>
 	 
<tr><td>10:30-11:30	<td><b>KEYNOTE: Sarah Tishkoff. <br><i><a href="keynotes.html#Tishkoff">African Evolutionary Genomics: Implications for human origins and adaptation </b></i></a>
 	 
<tr><td>11:30-11:50	<td>Ewa Szczurek and Niko Beerenwinkel. <br><i>Modeling mutual exclusivity of cancer mutations</i>
 	 
<tr><td>11:50-12:10	<td>Ben Raphael and Fabio Vandin. <br><i>Simultaneous Inference of Cancer Pathways and Tumor Progression from Cross-Sectional Mutation Data</i>
 	 
<tr><td>12:10-13:30	<td><b>LUNCH</b>

<tr><td>13:30-14:30	<td><b>Session 15. Chair: Teresa Przytycka, NIH</b>
 	 
<tr><td>13:30-13:50	<td>Keith Noto, Carla Brodley, Saeed Majidi, Diana Bianchi and Donna Slonim. <br><i>CSAX: Characterizing Systematic Anomalies in eXpression Data</i>
 	 
<tr><td>13:50-14:10	<td>Hua Wang, Heng Huang and Chris Ding. <br><i>Correlated Protein Function Prediction via Maximization of Data-Knowledge Consistency</i>
 	 
<tr><td>14:10-14:30	<td><b>HIGHLIGHT:</b> Julia Lasserre, Ho-Ryun Chung and Martin Vingron. <br><i>Finding Associations among Histone Modifications Using Sparse Partial Correlation Networks </i>
 	 
<tr><td>14:30-15:00	<td><b>BREAK</b>

<tr><td>15:00-16:00	<td><b>Session 16. Chair: Lenore Cowen, Tufts University </b>
 	 
<tr><td>15:00-15:20	<td>Marinka Zitnik and Blaz Zupan. <br><i>Imputation of Quantitative Genetic Interactions in Epistatic MAPs by Interaction Propagation Matrix Completion</i>
 	 
<tr><td>15:20-15:40	<td>Hamidreza Chitsaz and Mohammad Aminisharifabad. <br><i>Exact Learning of RNA Energy Parameters From Structure</i>
	
<tr><td>15:40-16:00	<td><b>HIGHLIGHT:</b> Raluca Gordan, Ning Shen, Iris Dror, Tianyin Zhou, John Horton, Remo Rohs and Martha Bulyk. <br><i>Genomic Regions Flanking E-Box Binding Sites Influence DNA Binding Specificity of bHLH Transcription Factors through DNA Shape</i>

<tr><td>16:00-16:30	<td><b>Award ceremony and closing remarks</b>
 	 
<tr><td>16:30	<td><b>Conference Adjourns</b>


