
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" >

  <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:700|Pontano+Sans' rel='stylesheet' type='text/css'>
  <link rel="stylesheet" type="text/css" href="course.css">
  <title>RECOMB 2014 - April 2-5, Pittsburgh, Pennsylvania</title>
</head>
<body>

<!-- ========================================================================= -->
<header id="masthead">
<img src="748x205_level_header_city2.JPG" width="935" height="256">
</header>

<!-- ========================================================================= -->
<aside id="sidebar">
    <section>
      <nav id="global">
       <ul>
          <li><a href="index.html">Home</li>
          <li><a href="geninfo.html">General Information</a></li>
          <li><a href="http://en.wikipedia.org/wiki/Alumni_Hall_%28University_of_Pittsburgh%29">Conference Venue</a></li>
          <li><a href="program.html">Program</a></li>
          <li><a href="registration.html">Registration</a></li>
          <li><a href="travel.html">Travel & Visa</a></li>
  <li><a href="lodging.html">Accommodations</a></li>
          <li><a href="travelfellowships.html">Travel Fellowships</a></li>
          <li><a href="keynotes.html">Keynote Speakers</a></li>
          <li><a href="acceptedpapers.html">Accepted Papers</a></li>
          <li><a href="posters.html">Poster Sessions</a></li>
<li><a href="keydates.html">Key Dates</a></li>
          <li><a href="satellites.html">Satellite Workshops</a></li>
          <li><a href="callforpapers.html">Call for Papers</a></li>
          <li><a href="callforposters.html">Call for Posters</a></li>
          <li><a href="callforhighlights.html">Call for Highlights</a></li>
          <li><a href="submission.html">Submission</a></li>
          <li><a href="about.html">About RECOMB</a></li>
          <li><a href="http://www.cmu.edu/ua/pittsburgh/index.html">About Pittsburgh</a></li>
          <li><a href="committees.html">Committees</a></li>
          <li><a href="sponsorship.html">Sponsorship Opportunities</a>
<li><a href="sponsors.html">Sponsors</a></li>
          <li><a href="contact.html">Contact</a></li>
        </ul>
      </nav>
    </section>
</aside>

<div id="content">

<!-- ========================================================================= -->



<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html><head><title>RECOMB-Seq 2014</title></head><body>



<h1>RECOMB-SEQ: FOURTH ANNUAL RECOMB



SATELLITE WORKSHOP ON MASSIVELY PARALLEL SEQUENCING</h1>







<p><b>March 31st - April 1st 2014<br>



Pittsburgh, PA, USA</b></p>



Rangos Hall, Carnegie Mellon University- <a href="https://mapsengine.google.com/map/edit?mid=z7bDiIx7WZL8.kmKKq7L9QKGw">Directions to Rangos Hall</a>

<p><b>Conference chairs: </b><a href="http://www.cbcb.umd.edu/%7Empop">Mihai Pop</a> (University of Maryland), <a href="http://research.mssm.edu/bashir/">Ali Bashir</a> (Mount Sinai School of Medicine)<b><br>



</b></p>

<center>
  <a href="recombseq.html#about">About</a>
  &nbsp;|&nbsp;
  <a href="recombseq.html#scope">Scope</a>
  &nbsp;|&nbsp;
  <a href="recombseq.html#program">Program and Accepted Papers</a>
  &nbsp;|&nbsp;
  <a href="recombseq.html#keynote">Keynote Speakers</a>
  &nbsp;|&nbsp;
  <a href="recombseq.html#steering">Steering Committee</a>
  &nbsp;|&nbsp;
  <a href="recombseq.html#progcom">Program Committee</a>
</center>

<h2>RECOMB-SEQ IS NOW IN SESSION!<BR>#RECOMBSEQ2014</H2>

<H2>Participate in the <a href="https://groups.google.com/forum/#!forum/recomb-seq-emerging-trends">Google Group discussion!</a></h2>

<h2><a name="about"></a>About the Workshop</h2>


<p> RECOMB-seq is a workshop held immediately before <a href="http://www.compbio.cmu.edu/recomb/index.html">the main RECOMB conference</a>, with a distinct paper submission process and
program. Our goal is to bring together the community of scientists
working on methods for the analysis of modern DNA sequencing data.</p>







<h2><a name="scope"></a><strong>Scope</strong><br>



</h2>







<p> The recent revolution in sequencing technology



has opened the door for myriads of new applications and bio-medical



discoveries. The genomes of thousands of human individuals have already



been sequenced and sequencing is increasingly becoming a part of



clinical diagnostics, especially in the context of cancer.&nbsp;



Furthermore, efforts are underway to map the genomic diversity on



earth, whether in vertebrate species (e.g., the Genome10K project), or



in the microbial ecosystems that



live in our bodies (e.g., the Human Microbiome project) and our



environment (e.g., the Earth Microbiome Project). The complexity and



sheer size of the data generated in such projects have highlighted the



limitations of current analytical methods. As technology continues to



evolve, we are faced with increasing challenges in managing,



processing, and extracting meaningful biological information from the



data being generated.&nbsp;<br>



 </p>







<p> We would like to invite contributions



describing new methodology aimed at dealing all aspects of massively



parallel sequencing data, including, but not limited to: </p>







<ul>







  <li>Translational applications of sequencing data, including cancer genomics and infectious disease diagnostics and surveillance<br>



</li>



  <li>Discovery and genotyping of genomic



variants, including SNPs, indels, and structural variants </li>



  <li>Local and de novo sequence assembly </li>



  <li>RNA sequencing, including the analysis of



RNA expression and novel transcript assembly </li>



  <li>Epigenetics, including ChIP-SEQ analysis,



methylation profiling, and histone modifications </li>



  <li>Metagenomics </li>



  <li>Read mapping </li>



  <li>Methods for emerging sequencing



technologies, such as single-cell or single molecule real time



sequencing</li>



</ul>




<h2><a name="program"></a><strong>Program and Accepted Papers</strong></h2>
<a href="recombseq2014program.pdf">Tentative Program for RECOMB-Seq 2014</a>


<h2><a name="keynote"></a><strong>Keynote Speakers<br></strong></h2>
<table>
<tr>
  <td><img src="lo_cropped.jpg" width="250"></td>
  <td> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</td>
  <td> <span style="font-weight: bold; text-decoration: underline;">Prof. Cecilia Lo</span> <br>F. Sargent Cheever Chair of the Department of Developmental Biology<br> University of Pittsburgh School of Medicine <br>
<a href="http://www.devbio.pitt.edu/subsites/lo/index.html">Homepage</a>
<br>
<p>
Prof. Lo is a world renowned developmental biologist who has
pioneered innovative approaches for studying developmental mechanisms
of congenital heart disease. Her work provides a compelling example of
how genomic technologies can be integrated with imaging, mutagenesis,
and other experimental approaches to uncover the basis of complex
developmental diseases. 
</p>
</td> 
</tr>
<tr>
  <td><img src="http://cs.brown.edu/people/faculty/ben_j._raphael.jpg" width="250"></td>
  <td> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</td>
  <td><span style="font-weight: bold; text-decoration: underline;">Prof. Ben Raphael </span> <br> Associate Professor, Department of Computer Science <br>
Director of the Center for Computational Molecular Biology <br> Brown University<br>
<a href="http://compbio.cs.brown.edu/">Homepage</a>
<br>
<p>

Prof. Raphael's  research is at the interface between computer science, mathematics, and biology.
He focuses on the design of combinatorial and statistical algorithms for the
interpretation of genomes. Particular areas of emphasis include
analysis of structural variation in human and cancer genomes, and network/pathway analysis of genetic variants.
He is the recipient of a Career Award at the Scientific Interface
from the Burroughs Wellcome Fund, an Alfred P. Sloan Research Fellowship, and a
National Science Foundation CAREER award.</p>
</td> 
</tr>
</table>

</p>







<ul>



</ul>



<h2>Important dates</h2>



Paper submission open: <span style="font-weight: bold;">September 20, 2013</span><br>



Regular paper submission deadline: <span style="font-weight: bold;">December 16, 2013</span><br>



Deadline for paper resubmitted from RECOMB: <span style="font-weight: bold;">December 16, 2013</span><br>



Paper acceptance notification: <span style="font-weight: bold;">January 23, 2014</span><br>



Paper final version due: <span style="font-weight: bold;">Feburary 13, 2014</span><br>



Emerging Trends/Poster submission open: <span style="font-weight: bold;">January 6, 2014</span><br>



Emerging Trends/Poster submission deadline: <span style="font-weight: bold;">February 7, 2014</span><br>



Poster acceptance notification: <span style="font-weight: bold;">February 14, 2014</span><br>







<ul>
</ul>







<h2><strong>Paper and Poster Submission</strong><br>







</h2>







<p> We are soliciting high-quality contributions



that will go through a rigorous peer-review process. Selected papers



will be presented at the workshop and published in a special



proceedings issue of BMC Bioinformatics. Please pay close attention to <a href="recombseq.html#MS">manuscript preparation instructions</a>.<br>



</p>



<p>In addition we are soliciting original contributions, including



significant research in progress to be presented as posters during



RECOMB-Seq.&nbsp;The accepted



poster abstracts will be published in the workshop book of abstracts.



Simultaneous submissions of the same poster to both RECOMB and



RECOMB-Seq are permitted. Please pay close attention to  <a href="recombseq.html#PS">poster preparation instructions</a>.</p>

<h3>Emerging trends session</h3>

This year we plan to organize a session on emerging trends in the use

of sequencing data in biomedical research.&nbsp; The goal of this

session is to provide a forum for research in progress in new

sequencing technologies and applications of sequencing technologies in

new application domains. Oral presentations for this session will be

selected from among the poster abstracts.<br>





<h3>Submission link</h3>



<p>Papers and poster abstracts can be submitted through EasyChair  <a href="http://www.easychair.org/conferences/?conf=recombseq2014">



http://www.easychair.org/conferences/?conf=recombseq2014</a></p>



<h3><strong>Special Handling of Papers Submitted to



RECOMB'2014</strong><br>







</h3>











<p>If you would like to consider submitting a



paper rejected by RECOMB'2014 to the satellite workshop, you may do so up to <span style="font-weight: bold;">December 16</span>(one

week after the RECOMB deadline). You must include in your

submission the reviews received at RECOMB together with a rebuttal

addressing the concerns raised by the reviewers.&nbsp; Ideally the

submitted manuscript must also have been modified according to the

reviews, or provide an explanation of why modifications are

unnecessary. If additional time is needed to make the necessary

changes,

please outline in your submission how you plan to address the

comments.&nbsp; Should your paper be accepted, an adequately revised

paper must be submitted by the full paper submission deadline. All

reviews will be treated confidentially and will only be

visible to the PC members evaluating your paper. <br>



</p>



<p>Please note that acceptance to RECOMB-Seq is not guaranteed even if



you have revised your submission.&nbsp; All papers, including those



submitted directly to RECOMB-Seq, will be equally evaluated on the



basis of scientific quality, technical novelty and level of interest to



the RECOMB-Seq community. <br>



</p>



<h3><a name="MS"></a>Manuscript preparation instructions<br>



 </h3>







<p> </p>







<p> The cover page should contain the title,



author(s) names and affiliations, an abstract, the keywords, and the



contact author e-mail. The manuscript should start with a succinct



statement of the problem, the results achieved, their significance, and



a comparison with previous work. This material should be understandable



to non-specialists. A technical exposition directed to the specialist



should follow. The length, excluding cover page and bibliography,



should not exceed 10 pages. The manuscript should be easy to read,



using at least 11 point font size on U.S. standard 8 1/2 by 11 inch



paper with no less than one inch margin all around. If the authors



believe that more details are absolutely necessary to substantiate the



claims of the paper, they may include a clearly marked appendix, which



might be read at the discretion of the reviewers. Manuscripts that



deviate significantly from these guidelines risk rejection without



consideration of their merits.</p>



<p>Also pay close attention to the <a href="Instructions&#32;for&#32;authors&#32;-&#32;BMC&#32;Appendix&#32;B.pdf">BMC manuscript preparation guidelines</a>. <br>



 </p>



<h3><strong><a name="PS"></a>Poster preparation instructions</strong><br>







</h3>







<p>Please submit a one-page abstract (about 200



words, plain justified text in at least 10pt character size, formatted



to fit A4: 21 x 29.7 cm) of your poster including title, author(s),



affiliation, e-mail and abstract text, via the easychair system. The



actual poster space will be 90 x 120 cm (width x height), which



corresponds to A0 format (or 16 pages in A4 format). <br>



 </p>




<h2><a name="steering"></a><strong>Steering Committee</strong></h2>
<ul>
<li>Cenk Sahinalp, Simon Fraser University</li>
<li>Eran Halperin, Tel Aviv University </li>
<li> Michael Brudno, University of Toronto</li>
<li> Ben Raphael, Brown University </li>
<li> Inanc Birol, Canada's Michael Smith Genome Sciences Centre</li>
</ul>

<p> </p>







<ul>

</ul>

<h2><a name="progcom"></a><strong>Program Committee</strong></h2>

<ul>
<li class="body_text">Todd Treangen, National Biodefense Analysis and Countermeasures Center</li>
<li class="body_text">Chun Ye, Broad Institute</li>
<li class="body_text">Manja Marz, University of Jena</li>
<li class="body_text">Ben Langmead, John Hopkins University</li>
<li class="body_text">Dominique Lavenier, INRIA</li>
<li class="body_text">Niranjan Nagarajan, Genome Institute of Singapore</li>
<li class="body_text">Laurent Noe, INRIA</li>
<li class="body_text">Pierre Peterlongo, INRIA</li>
<li class="body_text">Helene Touzet, INRIA</li>
<li class="body_text">Titus Brown, Michigan State University</li>
<li class="body_text">Xuegong Zhang, Tsinghua University</li>
<li class="body_text">Louxin Zhang, National University of Singapore</li>
<li class="body_text">Kai Tan, University of Iowa</li>
<li class="body_text">Jerome Waldispuhl, McGill University</li>
<li class="body_text">Lusheng Wang, City University of Hongkong</li>
<li class="body_text">Xiaohui Xie, UC Irvine</li>
<li class="body_text">Dong Xu, University of Missouri</li>
<li class="body_text">Arlindo Oliveira, INESC-ID</li>
<li class="body_text">Teresa Przytcka, NIH NCBI</li>
<li class="body_text">Ben Raphael, Brown University</li>
<li class="body_text">Marie-France  Sagot,INRIA</li>
<li class="body_text">Cenk Sahinalp, Simon Fraser University</li>
<li class="body_text">Steven Skiena, SUNY</li>
<li class="body_text">Andrew Smith, University of Southern California</li>
<li class="body_text">Thomas Lengauer, Max Planck Institute for Informatics</li>
<li class="body_text">Ming Li, University of Waterloo</li>
<li class="body_text">Jinze Liu, University of Kentucky</li>
<li class="body_text">Stefano Lonardi, University of California, Riverside</li>
<li class="body_text">Satoru Miyano, The University of Tokyo</li>
<li class="body_text">Bernard Moret, Swiss Federal Institutes of Technology</li>
<li class="body_text">William Noble, University of Washington</li>
<li class="body_text">Heng Huang, University of Texas</li>
<li class="body_text">Sorin Istrail, Brown University</li>
<li class="body_text">Rui Jiang, Tsinghua University</li>
<li class="body_text">Hyunju Lee, Gwangju Institute of Science and Technology</li>
<li class="body_text">Kun-Mao Chao, National Taiwan University</li>
<li class="body_text">Ting Chen, University of Southern California</li>
<li class="body_text">Francis Chin, University of Hong Kong</li>
<li class="body_text">Nadia El-Mabrouk, University of Montreal</li>
<li class="body_text">Joel Bader, Johns Hopkins University</li>
<li class="body_text">Dongbo Bu, Chinese Academy of Sciences</li>
<li class="body_text">Tatsuya Akutsu, Kyoto University</li>
<li class="body_text">Max Alekseyev, University of South Carolina</li>
<li class="body_text">Gang Fang, Mount Sinai School of Medicine</li>
<li class="body_text">Sergey Koren, National Biodefense Analysis and Countermeasures Center</li>
<li class="body_text">Haixu Tang, Indiana University</li>
<li class="body_text">Shaojie Zhang, University of Central Florida</li>
<li class="body_text">Vikas Bansal, Scripps Translational Science Institute</li>
<li class="body_text">Menachem Fromer, Mount Sinai School of Medicine</li>
<li class="body_text">Bojan Losic, Mount Sinai School of Medicine</li>
<li class="body_text">Adam Phillippy, National Biodefense Analysis and Countermeasures Center</li>
<li class="body_text">Michael Schatz, Cold Spring Harbor Laboratory</li>
<li class="body_text">Mark Chaisson, University of Washington</li>
</ul>














<p> </p>







<ul>



</ul>







</body></html>
